source/basic.tex

```diff
@@ -4059,8 +4058,8 @@
 \tcode{void}~(\ref{basic.types}) has three corresponding cv-qualified
 versions of its type: a \defn{const-qualified} version, a
 \defn{volatile-qualified} version, and a
-\defn{const-volatile-qualified} version. The term
-\term{object type}~(\ref{intro.object}) includes the cv-qualifiers
+\defn{const-volatile-qualified} version. An
+object's type~(\ref{intro.object}) includes the cv-qualifiers
```

Comment:
Yes, we're talking about the grammar element, although the entire paragraph is odd:
```
using T = const int;
T x1;
const int x2;
const T x3;
```
All of the x1, x2, x3 objects have the same const-qualified type, even though there is no "const" cv-qualifier in the decl-specifier-seq of x1. But that fix is for another day.

I've added the "grammarterm" and change the possessive.